### DISCUSSION TITLE
Error while generate code for MYSQL

### DISCUSSION THREAD

**Discussion by myeduate** [sqlc-dev/sqlc#1118] [Issue Triage] [answered]
i have query as below 

-- name: CreateCustomer :execresult
INSERT INTO  MST_CUSTOMER (
    cust_id, 
    cust_details, 
    cust_code,
    cust_addr, 
    cust_place, 
    cust_contact, 
    cust_tel, 
    cust_email,
    cust_url)
  VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?);

-- name: ListCustomers :many
SELECT * FROM MST_CUSTOMER  ORDER BY cust_addr

Bu when is generated, i see that last one character is missing.  from the case "cust_addr" "r" is missing 

const listCustomers = `-- name: ListCustomers :many
SELECT cust_id, cust_details, cust_code, cust_addr, cust_place, cust_contact, cust_tel, cust_email, cust_url FROM MST_CUSTOMER  ORDER BY cust_add

Also i tried with other variables from the table, its same.

`

**Comment by kyleconroy** [comment, marked as answer]
The workaround here is the add a semicolon at the end of the fileZ I need to update the MySQL parser to error when a query isn't terminated.

**Reply by myeduate** [reply]
thank you
